engine/src/test/scala/cromwell/engine/WorkflowStoreSpec.scala

```diff
@@ -0,0 +1,39 @@
+package cromwell.engine
+
+import cromwell.engine.workflow.WorkflowStore
+import cromwell.util.SampleWdl.HelloWorld
+import org.scalatest.{FlatSpec, Matchers}
+
+class WorkflowStoreSpec extends FlatSpec with Matchers with WorkflowStore {
```

Comment:
It's because I wanted the test to be a little whitebox-y but I didn't want what became the `dump` method to be public. I suppose what ended up here is in the worst of both worlds camp.